docker-compose.override.yml

```diff
@@ -40,6 +40,7 @@ services:
       dockerfile: Dockerfile
 
   processor:
+    user: root
```

Comment:
Do we really need to run as root for development? Seems like you chown everything to the node user that we would need. Is it because the mounted volumes are owned by root?

Reply:
chown'ing in the frontend is doable because we are using a multi-stage docker file, but in the processor, the same stage is used for production and development.

I made the user root in development mode because the node user doesn't own the `src` dir, so recompiling and reloading the server on file save doesn't work.

Reply:
Ok, lets fix that when we make processor Dockerfile multi-stage.